array_api_tests/__init__.py

```diff
@@ -1,11 +1,16 @@
 from functools import wraps
+from os import getenv
 
 from hypothesis import strategies as st
 from hypothesis.extra import array_api
 
+from . import _version
 from ._array_module import mod as _xp
 
-__all__ = ["xps"]
+__all__ = ["COMPLEX_VER", "api_version", "xps"]
+
+
+COMPLEX_VER: str = "2022.12"
```

Comment:
I don't like this. There's more that was added in 2022 than just complex numbers. 

Why not convert the version string to a tuple of numbers, so you can just compare like `if version >= (2022,)`.

Reply:
Ah this is just meant to represent the version where complex numbers were added, not some be-end-all. It existed as a variable for historic reasons, as it was previously `"draft"` and then something I'd just updated, but really I can grep-replace instances of it to `"2022.12"` now

Reply:
I would rather just use inline version checks with the actual version. That's much more explicit to anyone reading the test code. We can include links to the spec changelog or versionadded admonitions as references if needed.